### PR TITLE
Feaet: add sprite flip demo

### PR DESCRIPTION
**Pull request by singlecoder**
Add sprite flip demo

### CODE REVIEW

File: demos/sprite-flip/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+engine.resourceManager
+  .load<Texture2D>({
+    url: "https://gw.alipayobjects.com/mdn/rms_7c464e/afts/img/A*KjnzTpE8LdAAAAAAAAAAAAAAARQnAQ",
+    type: AssetType.Texture2D
+  })
+  .then((texture) => {
+    for (let i = 0; i < 4; ++i) {
+      const spriteEntity = rootEntity.createChild(`sprite_${i}`);
+      const { transform } = spriteEntity;
+      transform.setPosition(5 + 10 * (i - 2), 0, 0);
```
Comment (GuoLei1990) [inline demos/sprite-flip/index.ts:27]: simply code

**Review by GuoLei1990 — COMMENTED** [review]
You can warp a function and set position and flip as param

File: demos/sprite-flip/index.ts (hunk 1 above)
Comment (singlecoder) [reply demos/sprite-flip/index.ts]: > simply code

done

**Comment by singlecoder** [comment]
> You can warp a function and set position and flip as param

done

**Pull request merged**